CHANGELOG.next.asciidoc

```diff
@@ -92,6 +92,7 @@ https://github.com/elastic/beats/compare/v8.2.0\...main[Check the HEAD diff]
 - Fixing system tests not returning expected content encoding for azure blob storage input. {pull}34412[34412]
 - [Azure Logs] Fix authentication_processing_details parsing in sign-in logs. {issue}34330[34330] {pull}34478[34478]
 - Prevent Elasticsearch from spewing log warnings about redundant wildcard when setting up ingest pipelines. {issue}34249[34249] {pull}34550[34550]
+- Gracefully handle channel not found errors. {issue}30201[30201] {pull}34605[34605]
```

Comment:
I'd note that this refers to Windows even channels since in the context of filebeat, that may not be obvious.

Reply:
Fair point! I'll add a note for that.